powersimdata/network/usa_tamu/constants/zones.py

```diff
@@ -2,6 +2,28 @@
 
 import pandas as pd
 
+_exports = [
+    "abv",
+    "abv2interconnect",
+    "abv2loadzone",
+    "abv2state",
+    "id2abv",
+    "id2timezone",
+    "interconnect",
+    "interconnect2abv",
+    "interconnect2id",
+    "interconnect2loadzone",
+    "interconnect2timezone",
+    "interconnect_combinations",
+    "loadzone",
+    "loadzone2interconnect",
+    "loadzone2state",
+    "state",
+    "state2abv",
+    "state2loadzone",
+    "timezone2id",
+]
```

Comment:
Shall we add `id2zone` and `zone2id` here? They are attributes of grid object and we would like to access the info without loading a grid.

Reply:
I will do so

Reply:
Done. To follow the naming convention in `powersimdata.network.usa_tamu.zones`, I use `id2loadzone` and `loadzone2id`.